src/app/mork/celery/tasks/emailing.py

```diff
@@ -55,13 +55,14 @@ def warn_user(self, email: str, username: str, dry_run: bool = True):
     """Celery task that warns the specified user by sending an email."""
     # Check that user has not already received a warning email
     if check_email_already_sent(email):
-        raise EmailAlreadySent("An email has already been sent to this user")
+        logger.warning("An email has already been sent to this user")
+        return
```

Comment:
As we only rely on the `auth_user` table as our source of users to send an email to, we may enter in this condition a lot of times between two global `email_inactive_users` (and it would happen for each user having entries in a protected tables).
My proposed solution was to silently exit the task in this case 
I'm open to suggestion as I'm not sure how I would avoid raising that many errors.